packages/desktop-client/src/components/schedules/ScheduleLink.tsx

```diff
@@ -70,6 +90,16 @@ export function ScheduleLink({
           value={filter}
           onChange={setFilter}
         />
+        {ids.length === 1 && (
+          <Button
+            type="primary"
+            style={{ marginLeft: 15, padding: '4px 10px' }}
+            onClick={onCreate}
+          >
+            <SvgAdd style={{ width: '20px', padding: '3px' }} />
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
            <SvgAdd style={{ width: 20, padding: 3 }} />
```

Reply:
done